tests/common_fixtures.py

```diff
@@ -41,6 +42,9 @@ def config_test_base():
     mlrun.db._last_db_url = None
     mlrun.datastore.store_manager._db = None
 
+    # remove singletons in case they were change (we don't want changes to pass between tests)
```

Comment:
```suggestion
    # remove singletons in case they were changed (we don't want changes to pass between tests)
```

Reply:
Not relevant for this PR but it's an indicator these should probably be regular instantiated classes, instantiated in some initialization and not necessarily singletons

Reply:
Just to explain what happened here - one test was mocking one of the singleton's methods, and then this mock stayed there and affected other tests as well